### ISSUE TITLE
Error message could be more specific when calculate distances using collection ids without load

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#7036]
#### Steps/Code to reproduce:
```python
from pymilvus_orm import CollectionSchema, FieldSchema
from pymilvus_orm import Collection
from pymilvus_orm import connections
from pymilvus_orm.types import DataType
dim = 2
int64_field = FieldSchema(name="int64", dtype=DataType.INT64, is_primary=True)
float_field = FieldSchema(name="float", dtype=DataType.FLOAT)
float_vector = FieldSchema(name="float_vector", dtype=DataType.FLOAT_VECTOR, dim=dim)
schema = CollectionSchema(fields=[int64_field, float_field, float_vector])
connections.connect()
 import numpy as np
collection = Collection("test_search_collection_binbin_1", schema=schema)
collection.create_partition(partition_name="search_partition_1", description="search partition")
par = collection.partitions
nb = 1000
import random
vectors = [[random.random() for _ in range(dim)] for _ in range(nb)]
res = collection.insert([[i for i in range(nb)], [np.float32(i) for i in range(nb)], vectors],  par[0].name)
collection.insert([[i for i in range(nb)], [np.float32(i) for i in range(nb)], vectors],  par[1].name)
ids = list(res.primary_keys)
vectors_l = [[1, 2]]
op_l = {"float_vectors": vectors_l}
op_r = {"ids": ids, "collection": collection.name, "field": "float_vector"}
metric = "L2"
sqrt = False
params = {"metric": metric, "sqrt": sqrt}
from pymilvus_orm import utility
result = utility.calc_distance(vectors_left=op_l, vectors_right=op_r, params=params)
```

#### Expected result:
something like "*** was not loaded into memory"

#### Actual results:
Failed to fetch vectors

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93): bdb8396
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2):
Name: pymilvus
Version: 2.0.0rc3.dev16
Name: pymilvus-orm
Version: 2.0.0rc3.dev15
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

**Comment by yhmo** [comment]
Fixed by #7175

**Comment by binbinlv** [comment]
Verified and fixed.

Name: pymilvus
Version: 2.0.0rc5.dev20

Results:
<BaseException: (code=1, message=collection test_search_collection_binbin_1 was not loaded into memory)
